lighthouse-cli/test/smokehouse/smokehouse.js

```diff
@@ -116,15 +111,11 @@ function runLighthouse(url, configPath, isDebug) {
 
   let errorCode;
   let lhr = {requestedUrl: url, finalUrl: url, audits: {}};
-  if (runResults.status === PAGE_HUNG_EXIT_CODE) {
-    errorCode = 'PAGE_HUNG';
-  } else if (runResults.status === INSECURE_DOCUMENT_REQUEST_EXIT_CODE) {
-    errorCode = 'INSECURE_DOCUMENT_REQUEST';
-  } else {
+  if (fs.existsSync(outputPath)) {
```

Comment:
should we turn `isUnexpectedFatalResult` into `exitCode !== 0 || !fs.existsSync(outputPath)` and this will always exist by this point? Seems like no output is an unexpected failure, even if status code is 0

Reply:
Yeah good call 👍